apps/main/src/dao/components/PageGauges/GaugeListItem/index.tsx

```diff
@@ -119,7 +129,7 @@ const GaugeListItem = ({
             >
               {t`VISIT ${gaugeCurveApiData?.isPool ? 'POOL' : 'MARKET'}`}
             </ExternalLinkIconButton>
-            <InternalLinkButton to={`/gauges/${gaugeData.address}`}>{t`VISIT GAUGE`}</InternalLinkButton>
+            <InternalLinkButton to={`/gauges/${gaugeData.effective_address}`}>{t`VISIT GAUGE`}</InternalLinkButton>
```

Comment:
This kind of links are broken now, I thought I got rid of all of them!
```suggestion
            <InternalLinkButton to={getEthPath(`${DAO_ROUTES.GAUGES/${gaugeData.effective_address}`)}}>{t`VISIT GAUGE`}</InternalLinkButton>
```